added isEU (#27)
ipinfo/ruby: README.md

country = details.country # US
country_name = details.country_name # United States
```

> ##### EU Country

Reviewer: Newline separation:

```suggestion

##### European Union (EU) Country
```

> `details.is_eu` will return true if the country is a member of EU , as supplied by the `eu.json` file.

Reviewer: ```suggestion
`details.is_eu` will return `true` if the country is a member of the European Union (EU), as supplied by the `eu.json` file.
```

```ruby
is_eu = details.is_eu # false
```

> It is possible to change the file by setting the `eucountries` setting when creating the `IPinfo` object.

Reviewer: Missing an extra newline for separation and let's please rename the setting itself:

```suggestion

It is possible to change the file by setting the `eu_countries` setting when creating the `IPinfo` object.
```

> The file must be a `.json` file with the following [structure.](lib/ipinfo/eu.json)

Reviewer: ```suggestion
The file must be a `.json` file with the [following structure](lib/ipinfo/eu.json).
```
